app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -801,8 +803,17 @@ struct DeviceDelegateOpenXR::State {
     // Invoke the callback. We only clear it if the load has started. Otherwise we will retry later.
     // This happens for example if a profile with no 3D models like hand interaction is loaded on
     // start. If we clear the callback it won't ever be called in case the user grabs a controller.
-    if (controllersReadyCallback())
+    if (controllersReadyCallback()) {
+        MaybeNotifyControllersAvailable();
         controllersReadyCallback = nullptr;
+    }
+  }
+
+  void MaybeNotifyControllersAvailable() {
```

Comment:
This is so small and used once that I think we don't really need a new method for that

Reply:
Done